platform/BUILD.bazel

```diff
@@ -190,7 +190,14 @@ objc_library(
         "//platform/ios:ios_app_srcs",
         "//platform/darwin:app/CustomStyleLayerExample.h",
         "//platform/darwin:app/CustomStyleLayerExample.m",
-    ],
+    ] + select({
+        "//:metal_renderer": [
+        "//platform/darwin:app/WideLineStyleLayer.h",
```

Comment:
Why is the source for the iOS app in `darwin`?

Reply:
Can we add these to the `ios_app_srcs` target and not use `exports_files`?

`exports_files` makes it harder to reason about dependencies.

Reply:
That is where the original LimeGreenTriangle example was. We can move it to another location.

Reply:
Ah alright, makes sense.

Maybe you can try to avoid using `exports_files`, but I can have a look after it is merged as well.

Reply:
acknowledged! 👍

Reply:
I was trying to move the files to `ios` in the app and I realized the way it's done now is the `objc_library "iosapp"` is defined in `platform/` and it requires sources from `platform/ios` which needs the files in the `platform/ios` export list. The `ios_application` in `platform/ios` then adds the `//platform:iosapp` as dependency.